uint8 a
---
uint8[] barcode
